.github/workflows/checklist.yaml

```diff
@@ -79,6 +79,17 @@ jobs:
         run: |
           badge_url=$(python .github/scripts/create_badge.py "${{ env.file_name }}")
           echo "badge=$(echo $badge_url)" >> $GITHUB_ENV
+
+      # Generate url-safe, directly from checklist (env.name already altered)
+      - name: Generate URL-safe tool string
+        if: steps.check_labels.outputs.result == 'true'
+        run: |
+          tool_url=$(echo "$checklist" \
+            | jq -r '.name' \
+            | tr '[:upper:]' '[:lower:]' \
+            | sed 's/[^a-z0-9_ ]//g' \
+            | tr ' ' '-')
```

Comment:
Is this always the modification? Isn't there also a `.urls` we can pull directly from?